pkg/server/router/did.go

```diff
@@ -250,17 +253,24 @@ type GetDIDsRequest struct {
 // @Failure     400     {string} string "Bad request"
 // @Failure     500     {string} string "Internal server error"
 // @Router      /v1/dids [get]
-func (dr DIDRouter) GetDIDsByMethod(ctx context.Context, w http.ResponseWriter, _ *http.Request) error {
+func (dr DIDRouter) GetDIDsByMethod(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
 	method := framework.GetParam(ctx, MethodParam)
+	deleted := framework.GetQueryValue(r, DeletedParam)
 	if method == nil {
 		errMsg := "get DIDs by method request missing method parameter"
 		logrus.Error(errMsg)
 		return framework.NewRequestErrorMsg(errMsg, http.StatusBadRequest)
 	}
 
+	isDeleted, err := strconv.ParseBool(*deleted)
```

Comment:
probably need to handle the case where *deleted is nil (if someone does a getDids request without a query param)